Instagram/View Controllers/HomeViewController.h

```diff
@@ -9,7 +9,7 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
-@interface HomeViewController : UIViewController
+@interface HomeViewController : UIViewController 
```

Comment:
Careful of checking in changes like this. It seems like no big deal but it slows down reviews. Another reason is the 'blame'. Often people will go to the version control 'blame' to see who last changed the file with the hope of discovering the reason for the change. They will see this as the most recent change and it will be more work for them to dig into the commit they care about.